docs/resources/cluster.md

```diff
@@ -4,19 +4,52 @@ page_title: "rancher2_cluster Resource"
 
 # rancher2\_cluster Resource
 
-Provides a Rancher v2 Cluster resource. This can be used to create Clusters for Rancher v2 environments and retrieve their information.
+Provides a Rancher v2 Cluster resource. This can be used to create imported Clusters for Rancher v2 environments and retrieve their information.
```

Comment:
It looks like this can file also states it can be used to create [custom clusters](https://github.com/jiaqiluo/terraform-provider-rancher2/blob/update-doc/docs/resources/cluster.md#creating-rancher-v2-rke-cluster)? We should remove the other RKE1 examples from this file if we want to say that only imported clusters are supported - they don't work anymore anyway iiuc.